chart/epinio/templates/default-user.yaml

```diff
@@ -4,8 +4,25 @@ type: BasicAuth
 metadata:
   labels:
     epinio.suse.org/api-user-credentials: "true"
-  name: default-epinio-user
+    epinio.suse.org/role: "admin"
+  name: admin-epinio-user
   namespace: {{ .Release.Namespace }}
 stringData:
-  password: {{ .Values.api.password }}
   username: {{ .Values.api.username }}
+  password: {{ .Values.api.password }}
+---
+apiVersion: v1
+kind: Secret
+type: BasicAuth
+metadata:
+  labels:
+    epinio.suse.org/api-user-credentials: "true"
+    epinio.suse.org/role: "namespaced-user"
+  name: default-epinio-user
+  namespace: {{ .Release.Namespace }}
+stringData:
+  username: epinio
+  password: password
+  namespaces: |
+    workspace
+    workspace2
```

Comment:
What is `workspace2` ?

Reply:
Oh, well, it was just a test with more namespaces, but yes, since we are not creating it I would remove it!